pkg/services/apiserver/storage/entity/selector.go

```diff
@@ -48,11 +45,6 @@ func ReadLabelSelectors(selector labels.Selector) (Requirements, labels.Selector
 				return requirements, newSelector, apierrors.NewBadRequest(SortByKey + " label selector only supports in")
 			}
 			requirements.SortBy = r.Values().List()
-		case utils.AnnoKeyOriginKey:
```

Comment:
I don't think we need this now -- if it is somehow making things work for playlists dual writer, that is a fluke and not going to work in general.